include/CL/opencl.hpp

```diff
@@ -10253,7 +10373,102 @@ namespace compatibility {
     };
 } // namespace compatibility
 
+#if CL_HPP_TARGET_OPENCL_VERSION >= 300
+class SemaphoreKhr : public detail::Wrapper<cl_semaphore_khr>
```

Comment:
Bikeshedding: Should this be `SempahoreKhr` or `SemaphoreKHR` (or maybe even just `Semaphore`)?

I think I'd prefer just `Semaphore` personally...

Reply:
Done, thank you! I've used `Semaphore`.